### PR TITLE
parallel quicksort example

### PR DESCRIPTION
**Pull request by helloshuangzi**

### CODE REVIEW

**Comment by daiyip** [comment]
Please add a example link in README.md and wiki.

**Comment by fs-eire** [comment]
how about to put the change of transport.md in another PR? It helps to track the change history.

File: examples/tutorial/parallel-quick-sort/parallel-quick-sort.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+    // Execute parallelQuickSort.
+    let parallelLength = length / NUMBER_OF_WORKERS;
+    let start2 = Date.now();
+    return zone.execute(parallelQuickSort, [ta2, 0, length - 1, parallelLength]).then(result => {
+        console.log('parallelQuickSort:\tIt took (', Date.now() - start2, ') MS to sort an array with ', length, ' elements.');
+
+        // Assert the TypedArray has been sorted in ascending order.
+        for (let i = 0; i < length - 1; i++) {
+            assert(ta2[i] <= ta2[++i], 'the array is not in ascending order after quicksort.')
```
Comment (fs-eire) [inline examples/tutorial/parallel-quick-sort/parallel-quick-sort.js]: do you mean `ta2[i] <= ta2[i+1]`

Comment (helloshuangzi) [reply examples/tutorial/parallel-quick-sort/parallel-quick-sort.js]: Good catch

**Pull request merged**